vision-component-archetype/src/main/resources/archetype-resources/__rootArtifactId__-build/pom.xml

```diff
@@ -57,8 +57,8 @@
                     <moduleName>${project.parent.name}</moduleName>
                     <moduleDescription>${project.description}</moduleDescription>
                     <moduleVersion>${version}</moduleVersion>
-                    <requiredIgnitionVersion>7.7.0</requiredIgnitionVersion>
-                    <requiredFrameworkVersion>6</requiredFrameworkVersion>
+                    <requiredIgnitionVersion>7.8.0</requiredIgnitionVersion>
+                    <requiredFrameworkVersion>7</requiredFrameworkVersion>
```

Comment:
Tried to do this, but I hit an error when building the generated module: [ERROR] Failed to execute goal com.inductiveautomation.ignitionsdk:ignition-maven-plugin:1.0.11:modl (default) on project my-module-build: The parameters 'requiredFrameworkVersion' for goal com.inductiveautomation.ignitionsdk:ignition-maven-plugin:1.0.11:modl are missing or invalid -> [Help 1]